lib/experimental/fonts/font-library/class-wp-font-family.php

```diff
@@ -541,6 +542,7 @@ private function create_or_update_font_post() {
 	 * @return array|WP_Error An array of font family data on success, WP_Error otherwise.
 	 */
 	public function install( $files = null ) {
+		add_filter( 'upload_mimes', array( 'WP_Font_Library', 'set_allowed_mime_types' ) );
 		add_filter( 'upload_dir', array( 'WP_Font_Library', 'set_upload_dir' ) );
 		$were_assets_written = $this->download_or_move_font_faces( $files );
 		remove_filter( 'upload_dir', array( 'WP_Font_Library', 'set_upload_dir' ) );
```

Comment:
Should we remove the filter as we do with the `upload_dir`? I would believe that yes, because otherwise, we would be enabling that globally and that's not the intention I guess.

Reply:
I tried removing that and the mime type function is no longer working.  I'm not sure that we can.

Reply:
I believe that we should bring this in as it stands.  Testing showed we may need the global filter, but the local filter (get_upload_overrides()) maybe not.

Either way I believe this should be brought in and we can work out if it makes sense to pare it down.

Reply:
Ok, seems good to go. If the mimes in `get_upload_overrides()` are not necessary, we could remove them, right? Could you add a follow-up issue to remind us to do that?

Reply:
Agreed. Removed it here #54647